tools/c2chapel/test/enum.h

```diff
@@ -0,0 +1,32 @@
+
+/* Header file for enum test script */
+/* Enum without values */
+enum 
+{
+  TEST_STATUS_PASSED,
+  TEST_STATUS_FAILED
+};
+
+/* Enum with values */
+enum 
+{
+  PASSED = 1,
+  FAILED = 2
+};
+
+typedef enum 
+{
+  ENUM_ERROR = 1,
+  ENUM_WVALUES_ERROR = 2,
+  TD_ENUM_ERROR = 3,
+  FUNC_ENUM_ERROR = 4,
+  STRUCT_ENUM_ERROR = 5
```

Comment:
@benharsh the if (type) was getting passed as the typedef'd enums are a TypeDecl class not a Decl class. Added method genTypeEnum() to fix.